pkg/tfgen/generate.go

```diff
@@ -684,6 +684,16 @@ func (g *generator) gatherDataSource(rawname string,
 		}
 	}
 
+	// Add the special "id" attribute. This isn't exposed via terraform schema, so we make one up.
+	sch := &schema.Schema{
+		Type: schema.TypeString,
+		Computed: true,
+	}
+	cust := &tfbridge.SchemaInfo{}
+	rawdoc := "id is the provider-assigned unique ID for this managed resource. It is set during deployments and may be missing (undefined) during planning phases."
```

Comment:
I believe we can leave out the second sentence, since AFAIK data sources will always return true IDs (unlike resources that we are provisioning, where the ID might not be known during planning/previews).

Reply:
Sounds right, happy to change. Do you as a team have a preference for doing changes like that via another commit in the PR, or an amended commit?

Reply:
Slight preference for amended to keep commit history clean, but I'm happy either way :smile: